app/views/shared/forms/_applicant_form.html.erb

```diff
@@ -14,7 +14,8 @@
       ) do %>
 
     <div class="govuk-!-padding-bottom-2"></div>
```

Comment:
CYC: Can we remove these empty "spacer divs" from this view?

Reply:
As @RoseSAK is off until next week, do you want to add a commit and I can approve it?

Reply:
Second commit added @colinbruce and @RoseSAK